### DISCUSSION TITLE
Wiki, Windows client wiki nonexistent

### DISCUSSION THREAD

**Discussion by covelus** [cmu-sei/GHOSTS#54] [Q&A] [answered]
I would like to point out that, in the Wiki, when I click in "Installation: Windows Client", which links to https://github.com/cmu-sei/GHOSTS/wiki/Installing-the-Windows-Client, actually it redirects me to the same wiki page, and says that I "do not have permission to update this wiki" (so, I guess it's trying to create an nonexistent page)...

![image](https://user-images.githubusercontent.com/14851622/106265713-b8e20980-6227-11eb-8055-f92717923165.png)

I suggest fixing this or link to https://github.com/cmu-sei/GHOSTS/wiki/Installation-from-distribution-binaries
Thank you

**Comment by sei-dupdyke** [comment, marked as answer]
Hello, I fixed the url in the navigation, sorry about that. It should be:

https://github.com/cmu-sei/GHOSTS/wiki/Installing-The-Windows-Client

Thank you.

**Reply by covelus** [reply]
Thank you very much
